src/sentry/api/endpoints/setup_wizard.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+import logging
+
+from rest_framework.response import Response
+
+from sentry import ratelimits
+from django.core.cache import cache
```

Comment:
Since this is a permanent thing, let's not use this cache. This cache is ephemeral and won't error if it's down, so things will just silently fail.

Should use `sentry.cache:default_cache` instead, which is backed by redis.